django.utils.autoreload INFO 2024-05-29 19:22:51,041 autoreload 1896 140641658416000 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 19:23:07,771 log 1896 140641598793280 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/smartif.py", line 179, in translate_token
    op = OPERATORS[token]
KeyError: 'user.has_change_permission()'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/loader.py", line 42, in select_template
    return engine.get_template(template_name)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/defaulttags.py", line 953, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/defaulttags.py", line 952, in do_if
    condition = TemplateIfParser(parser, bits).parse()
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/defaulttags.py", line 885, in __init__
    super().__init__(*args, **kwargs)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/smartif.py", line 170, in __init__
    mapped_tokens.append(self.translate_token(token))
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/smartif.py", line 181, in translate_token
    return self.create_var(token)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/defaulttags.py", line 888, in create_var
    return TemplateLiteral(self.template_parser.compile_filter(value), value)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/base.py", line 600, in compile_filter
    return FilterExpression(token, self)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/template/base.py", line 703, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '()' from 'user.has_change_permission()'
django.request WARNING 2024-05-29 19:27:58,064 log 1896 140641598793280 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-05-29 19:40:00,557 autoreload 1896 140641658416000 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:40:02,492 autoreload 2935 139624267553664 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:40:07,664 autoreload 2935 139624267553664 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:40:09,295 autoreload 2980 140304199928704 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:40:11,936 autoreload 2980 140304199928704 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:40:13,659 autoreload 2994 140067689671552 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:40:27,694 autoreload 2994 140067689671552 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:40:29,653 autoreload 3027 140407285218176 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:40:31,294 autoreload 3027 140407285218176 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:40:32,965 autoreload 3051 140149662460800 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:40:42,467 autoreload 3051 140149662460800 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:40:44,466 autoreload 3093 140195994016640 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:40:49,604 autoreload 3093 140195994016640 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:40:51,382 autoreload 3138 139943073856384 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:41:02,101 autoreload 3138 139943073856384 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:41:04,122 autoreload 3162 139888956124032 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:41:09,561 autoreload 3162 139888956124032 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:41:11,241 autoreload 3207 139825913588608 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:41:13,286 autoreload 3207 139825913588608 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:41:15,157 autoreload 3221 140452946271104 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:41:19,579 autoreload 3221 140452946271104 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:41:21,478 autoreload 3254 140025544182656 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:41:25,834 autoreload 3254 140025544182656 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:41:27,558 autoreload 3278 140460622879616 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:48:40,288 autoreload 3278 140460622879616 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:48:42,471 autoreload 3518 140042028710784 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:48:44,161 autoreload 3518 140042028710784 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:48:45,966 autoreload 3542 139665840376704 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:48:55,801 autoreload 3542 139665840376704 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:48:57,577 autoreload 3587 140208949263232 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 19:49:03,462 log 3587 140208889640512 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/104
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 141, in customer_view_notes
    customer_note_form = CustomerNoteForm(initial={"note": customer_note.note})
UnboundLocalError: local variable 'customer_note' referenced before assignment
django.request WARNING 2024-05-29 19:49:03,977 log 3587 140208889640512 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-05-29 19:49:18,415 autoreload 3587 140208949263232 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:49:20,462 autoreload 3655 140185336626048 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:49:24,821 autoreload 3655 140185336626048 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:49:26,785 autoreload 3693 139869555649408 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:49:35,055 autoreload 3693 139869555649408 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:49:37,041 autoreload 3723 139785159871360 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:49:44,671 autoreload 3723 139785159871360 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:49:46,667 autoreload 3769 140130146433920 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:49:51,167 autoreload 3769 140130146433920 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:49:52,877 autoreload 3804 139839852112768 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 19:49:58,317 log 3804 139839792490048 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/104
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 126, in customer_view_notes
    customer_note_text = customer_note.note
UnboundLocalError: local variable 'customer_note' referenced before assignment
django.request WARNING 2024-05-29 19:49:59,347 log 3804 139839792490048 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-05-29 19:50:19,563 autoreload 3804 139839852112768 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:50:21,593 autoreload 3860 140418378247040 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:50:24,798 autoreload 3860 140418378247040 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:50:26,650 autoreload 3884 139659783535488 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:50:42,075 autoreload 3884 139659783535488 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:50:44,079 autoreload 3929 139879787973504 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:50:46,187 autoreload 3929 139879787973504 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:50:47,879 autoreload 3953 139929684753280 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:50:52,386 autoreload 3953 139929684753280 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:50:54,182 autoreload 3977 140396354669440 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:50:57,560 autoreload 3977 140396354669440 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:50:59,548 autoreload 4001 140279459425152 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:51:09,283 autoreload 4001 140279459425152 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:51:11,184 autoreload 4046 140500463897472 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:51:13,354 autoreload 4046 140500463897472 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:51:15,045 autoreload 4070 140512486034304 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:51:48,025 autoreload 4070 140512486034304 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:51:49,963 autoreload 4115 139967687338880 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:51:55,264 autoreload 4115 139967687338880 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:51:57,146 autoreload 4160 139976225876864 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:52:00,653 autoreload 4160 139976225876864 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:52:02,435 autoreload 4184 139627715160960 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:52:09,410 autoreload 4184 139627715160960 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:52:11,282 autoreload 4219 140371367844736 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 19:52:54,985 log 4219 140371308222016 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/104
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 132, in customer_view_notes
    editname = " By : "+request.user+")"
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/utils/functional.py", line 268, in inner
    return func(_wrapped, *args)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/utils/functional.py", line 452, in __radd__
    return other + self
TypeError: can only concatenate str (not "User") to str
django.request WARNING 2024-05-29 19:52:55,761 log 4219 140371308222016 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-05-29 19:53:18,253 autoreload 4219 140371367844736 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:53:20,179 autoreload 4338 140422154980224 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 19:53:29,477 log 4338 140422095357504 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/104
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 132, in customer_view_notes
    editname = " By : "+request.user.name+")"
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/utils/functional.py", line 268, in inner
    return func(_wrapped, *args)
AttributeError: 'User' object has no attribute 'name'
django.utils.autoreload INFO 2024-05-29 19:53:49,782 autoreload 4338 140422154980224 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:53:51,761 autoreload 4421 140299594849152 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 19:54:28,547 log 4421 140299468011072 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/104
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.NotNullViolation: null value in column "created_by_id" of relation "customers_customernote" violates not-null constraint
DETAIL:  Failing row contains (105, <p>Test<b>&nbsp;Test </b><i style="">Hello</i><i></i></p>(Edited..., 2024-05-29 19:54:28.131613+00, null, null).


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 136, in customer_view_notes
    customer_note.save()
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.IntegrityError: null value in column "created_by_id" of relation "customers_customernote" violates not-null constraint
DETAIL:  Failing row contains (105, <p>Test<b>&nbsp;Test </b><i style="">Hello</i><i></i></p>(Edited..., 2024-05-29 19:54:28.131613+00, null, null).

django.utils.autoreload INFO 2024-05-29 19:54:51,351 autoreload 4421 140299594849152 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:54:53,302 autoreload 4569 139899390761856 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:54:55,423 autoreload 4569 139899390761856 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:54:57,079 autoreload 4581 140326573292416 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:55:17,906 autoreload 4581 140326573292416 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:55:20,078 autoreload 4626 140010077211520 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:55:22,192 autoreload 4626 140010077211520 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:55:23,876 autoreload 4650 140269181107072 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 19:55:47,457 log 4650 140269121484352 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/104
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 135, in customer_view_notes
    customer_note.pk = id
NameError: name 'customer_note' is not defined
django.request WARNING 2024-05-29 19:55:48,047 log 4650 140269121484352 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-05-29 19:56:04,201 autoreload 4650 140269181107072 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:56:06,056 autoreload 4774 140110090972032 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:56:08,388 autoreload 4774 140110090972032 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:56:10,375 autoreload 4798 139784821664640 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:56:17,988 autoreload 4798 139784821664640 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:56:19,599 autoreload 4840 139673691429760 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 19:56:30,469 log 4840 139673631807040 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/104
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 136, in customer_view_notes
    customer_note_get.note = ''.join([customer_note.note, editdate, editname])
NameError: name 'customer_note' is not defined
django.utils.autoreload INFO 2024-05-29 19:57:17,186 autoreload 4840 139673691429760 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:57:19,702 autoreload 4923 140451670059904 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:57:25,157 autoreload 4923 140451670059904 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:57:26,802 autoreload 4968 139774366833536 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:59:40,886 autoreload 4968 139774366833536 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:59:42,798 autoreload 5010 140643552373632 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:59:44,912 autoreload 5010 140643552373632 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:59:46,690 autoreload 5034 140709378886528 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:59:49,056 autoreload 5034 140709378886528 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:59:50,969 autoreload 5058 140051406809984 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 19:59:54,209 autoreload 5058 140051406809984 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 19:59:56,064 autoreload 5082 140305865644928 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:00:02,694 autoreload 5082 140305865644928 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:00:04,588 autoreload 5117 140183494724480 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:00:06,664 autoreload 5117 140183494724480 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:00:08,447 autoreload 5135 140391326768000 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:00:36,708 autoreload 5135 140391326768000 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:00:38,660 autoreload 5172 140337851284352 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:00:47,386 autoreload 5172 140337851284352 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:00:49,590 autoreload 5217 140156195863424 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:02:57,268 autoreload 5217 140156195863424 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:02:59,296 autoreload 5262 139720666332032 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:03:03,095 autoreload 5262 139720666332032 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:03:05,175 autoreload 5288 140542491114368 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:03:12,821 autoreload 5288 140542491114368 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:03:14,986 autoreload 5331 139745448213376 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:03:18,307 autoreload 5331 139745448213376 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:03:20,148 autoreload 5355 139736884202368 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 20:03:41,240 log 5355 139736757302848 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/104
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 132, in customer_view_notes
    newnote = customer_note_form.note
AttributeError: 'CustomerNoteForm' object has no attribute 'note'
django.utils.autoreload INFO 2024-05-29 20:05:14,871 autoreload 5355 139736884202368 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:05:16,705 autoreload 5465 140107212389248 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:05:21,044 autoreload 5465 140107212389248 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:05:23,005 autoreload 5500 140606106364800 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:05:25,097 autoreload 5500 140606106364800 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:05:26,982 autoreload 5512 139666146859904 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:05:31,570 autoreload 5512 139666146859904 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:05:33,403 autoreload 5547 140640140929920 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 20:05:46,788 log 5547 140640081307200 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/104
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 132, in customer_view_notes
    newnote = customer_note_form.note
AttributeError: 'CustomerNoteForm' object has no attribute 'note'
django.utils.autoreload INFO 2024-05-29 20:06:25,656 autoreload 5547 140640140929920 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:06:27,687 autoreload 5642 140256325217152 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:06:34,229 autoreload 5642 140256325217152 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:06:35,965 autoreload 5684 139629079751552 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 20:06:44,660 log 5684 139629020653120 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/104
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 132, in customer_view_notes
    newnote = customer_note_form.note
AttributeError: 'CustomerNoteForm' object has no attribute 'note'
django.utils.autoreload INFO 2024-05-29 20:07:04,192 autoreload 5684 139629079751552 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:07:05,986 autoreload 5752 140299168480128 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:07:08,255 autoreload 5752 140299168480128 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:07:10,068 autoreload 5776 139893100833664 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:07:12,459 autoreload 5776 139893100833664 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:07:14,296 autoreload 5800 140694757493632 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:07:17,622 autoreload 5800 140694757493632 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:07:19,389 autoreload 5824 139857364364160 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:07:21,600 autoreload 5824 139857364364160 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:07:23,361 autoreload 5848 140138535582592 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:07:29,961 autoreload 5848 140138535582592 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:07:31,787 autoreload 5872 139695187229568 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 20:07:46,577 log 5872 139695127606848 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/104
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 132, in customer_view_notes
    newnote = request.note
AttributeError: 'WSGIRequest' object has no attribute 'note'
django.utils.autoreload INFO 2024-05-29 20:08:40,694 autoreload 5872 139695187229568 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:08:42,567 autoreload 5958 140082749217664 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:08:49,193 autoreload 5958 140082749217664 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:08:50,960 autoreload 6004 139639852342144 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 20:09:00,400 log 6004 139639721555520 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/104
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 132, in customer_view_notes
    newnote = data.note
NameError: name 'data' is not defined
django.utils.autoreload INFO 2024-05-29 20:09:33,551 autoreload 6004 139639852342144 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:09:35,462 autoreload 6071 140239132076928 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:09:38,724 autoreload 6071 140239132076928 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:09:40,692 autoreload 6095 139770254924672 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:09:49,398 autoreload 6095 139770254924672 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:09:51,384 autoreload 6140 140036729961344 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:09:53,457 autoreload 6140 140036729961344 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:09:55,310 autoreload 6164 140647795342208 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:10:01,801 autoreload 6164 140647795342208 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:10:03,342 autoreload 6206 139661069200256 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 20:10:24,413 log 6206 139661009577536 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/104
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 132, in customer_view_notes
    newnote = request.POST.note
AttributeError: 'QueryDict' object has no attribute 'note'
django.utils.autoreload INFO 2024-05-29 20:11:10,757 autoreload 6206 139661069200256 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:11:12,675 autoreload 6289 140498429565824 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:12:34,814 autoreload 6289 140498429565824 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:12:36,592 autoreload 6334 139680472750976 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 20:12:49,905 log 6334 139680413128256 Internal Server Error: /customers/8e05b134-0b51-46e4-ac9d-2e725f265a9b/customer_notes_list/104
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 145, in customer_view_notes
    MyModel.objects.filter(pk=id).update(val=F("val") + 1)
NameError: name 'MyModel' is not defined
django.utils.autoreload INFO 2024-05-29 20:13:05,039 autoreload 6334 139680472750976 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:13:06,882 autoreload 6436 139718401194880 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:13:29,625 autoreload 6436 139718401194880 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:13:31,549 autoreload 6500 140547580275584 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:13:39,203 autoreload 6500 140547580275584 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:13:41,066 autoreload 6545 140414215551872 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:13:53,060 autoreload 6545 140414215551872 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:13:54,508 autoreload 6590 139859819867008 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:14:28,194 autoreload 6590 139859819867008 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:14:29,896 autoreload 6632 140070088207232 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:14:36,485 autoreload 6632 140070088207232 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:14:38,272 autoreload 6677 139909796121472 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:15:05,275 autoreload 6677 139909796121472 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:15:07,150 autoreload 6722 140520217078656 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:17:10,270 autoreload 6722 140520217078656 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:17:12,359 autoreload 7063 140157356780416 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:17:20,092 autoreload 7063 140157356780416 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:17:21,757 autoreload 7109 140549885524864 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:17:27,370 autoreload 7109 140549885524864 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:17:29,163 autoreload 7154 139846603361152 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:17:31,260 autoreload 7154 139846603361152 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:17:33,193 autoreload 7168 139752445397888 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:17:39,791 autoreload 7168 139752445397888 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:17:41,610 autoreload 7201 140046430710656 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:17:48,363 autoreload 7201 140046430710656 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:17:50,166 autoreload 7246 140259306490752 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:18:03,197 autoreload 7246 140259306490752 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:18:05,177 autoreload 7288 139949383236480 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:18:11,787 autoreload 7288 139949383236480 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:18:13,514 autoreload 7333 140014045006720 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:18:16,802 autoreload 7333 140014045006720 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:18:18,487 autoreload 7357 140385897257856 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:18:20,662 autoreload 7357 140385897257856 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:18:22,580 autoreload 7381 140363738934144 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:18:24,667 autoreload 7381 140363738934144 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:18:26,756 autoreload 7395 139926484237184 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:20:22,939 autoreload 7395 139926484237184 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:20:25,085 autoreload 7446 139888966232960 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:22:16,522 autoreload 7446 139888966232960 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:22:18,313 autoreload 7491 140527574764416 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:22:20,478 autoreload 7491 140527574764416 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:22:22,255 autoreload 7515 140076303608704 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:22:24,470 autoreload 7515 140076303608704 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:22:26,540 autoreload 7533 139847203023744 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:22:49,592 autoreload 7533 139847203023744 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:22:51,508 autoreload 7586 140351835708288 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:22:58,154 autoreload 7586 140351835708288 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:23:00,068 autoreload 7631 140494122765184 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:23:13,468 autoreload 7631 140494122765184 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:23:15,468 autoreload 7676 140622505053056 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:23:26,704 autoreload 7676 140622505053056 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:23:28,783 autoreload 7721 139983821069184 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:23:38,879 autoreload 7721 139983821069184 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:23:40,967 autoreload 7778 140364604136320 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:23:59,565 autoreload 7778 140364604136320 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:24:01,690 autoreload 7831 140435150101376 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:24:06,147 autoreload 7831 140435150101376 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:24:08,088 autoreload 7869 139831390448512 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 20:24:45,987 log 7869 139831330825792 Internal Server Error: /customers/c7d83c33-a064-4016-a16d-4004b9120002/customer_notes_list/86
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 149, in customer_view_notes
    return redirect("customer_notes_list", customer_token=customer_token)
NameError: name 'redirect' is not defined
django.request WARNING 2024-05-29 20:24:46,410 log 7869 139831330825792 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-05-29 20:25:28,314 autoreload 7869 139831390448512 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:25:30,369 autoreload 8066 140620133444480 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 20:25:55,881 log 8066 140620073821760 Internal Server Error: /customers/c7d83c33-a064-4016-a16d-4004b9120002/customer_notes_list/86
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 150, in customer_view_notes
    return redirect("customer_notes_list", customer_token=customer_token)
NameError: name 'redirect' is not defined
django.request WARNING 2024-05-29 20:25:56,555 log 8066 140620073821760 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-05-29 20:29:09,586 autoreload 8066 140620133444480 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:29:11,874 autoreload 8202 139953463298944 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:29:16,081 autoreload 8202 139953463298944 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:29:17,979 autoreload 8237 139835843881856 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:29:24,275 autoreload 8237 139835843881856 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:29:26,167 autoreload 8278 140325615807360 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:29:47,958 autoreload 8278 140325615807360 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:29:49,966 autoreload 8323 140387451399040 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:30:04,323 autoreload 8323 140387451399040 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:30:06,585 autoreload 8368 140496957664128 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:30:12,075 autoreload 8368 140496957664128 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:30:13,908 autoreload 8413 139932550163328 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:30:16,208 autoreload 8413 139932550163328 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:30:18,181 autoreload 8437 140312154536832 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:30:20,540 autoreload 8437 140312154536832 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:30:22,600 autoreload 8451 140257597840256 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:30:29,270 autoreload 8451 140257597840256 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:30:31,092 autoreload 8484 140686827498368 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:30:35,403 autoreload 8484 140686827498368 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:30:37,263 autoreload 8519 139793970932608 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:30:41,804 autoreload 8519 139793970932608 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:30:43,683 autoreload 8549 140671043095424 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:30:52,679 autoreload 8549 140671043095424 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:30:54,794 autoreload 8594 139976553708416 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:30:57,241 autoreload 8594 139976553708416 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:30:58,986 autoreload 8618 139909782625152 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:31:01,245 autoreload 8618 139909782625152 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:31:03,240 autoreload 8642 140142273833856 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 20:31:17,311 log 8642 140142195852864 Internal Server Error: /customers/c7d83c33-a064-4016-a16d-4004b9120002/customer_notes_list/86
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 154, in customer_view_notes
    reverse("customer_notes_list", args=[customer_token]))
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/urls/base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/urls/resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'customer_notes_list' not found. 'customer_notes_list' is not a valid view function or pattern name.
django.request WARNING 2024-05-29 20:31:17,868 log 8642 140142195852864 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-05-29 20:32:22,383 autoreload 8642 140142273833856 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:32:24,383 autoreload 8732 140467280915328 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:32:36,467 autoreload 8732 140467280915328 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:32:38,542 autoreload 8777 140673264569216 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:32:46,385 autoreload 8777 140673264569216 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:32:48,278 autoreload 8822 139992631524224 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:32:50,494 autoreload 8822 139992631524224 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:32:52,345 autoreload 8846 140620004711296 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:32:55,235 autoreload 8846 140620004711296 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:32:57,168 autoreload 8870 139693415652224 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:34:18,479 autoreload 8870 139693415652224 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:34:20,556 autoreload 8920 140046006901632 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:35:32,107 autoreload 8920 140046006901632 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:35:34,152 autoreload 8965 139994386262912 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:35:35,923 autoreload 8965 139994386262912 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:35:37,640 autoreload 8989 139859828874112 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:35:47,301 autoreload 8989 139859828874112 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:35:49,281 autoreload 9034 140325415988096 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:35:52,149 autoreload 9034 140325415988096 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:35:54,065 autoreload 9058 139629816130432 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:36:05,080 autoreload 9058 139629816130432 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:36:07,180 autoreload 9100 140257217584000 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 20:36:22,056 log 9100 140257157961280 Internal Server Error: /customers/c7d83c33-a064-4016-a16d-4004b9120002/customer_notes_list/86
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 151, in customer_view_notes
    reverse("customer_notes_list", args=[customer_token]))
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/urls/base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/urls/resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'customer_notes_list' not found. 'customer_notes_list' is not a valid view function or pattern name.
django.utils.autoreload INFO 2024-05-29 20:37:27,301 autoreload 9100 140257217584000 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:37:29,284 autoreload 9212 140237187877760 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:37:31,450 autoreload 9212 140237187877760 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:37:33,217 autoreload 9236 140438883253120 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:37:37,813 autoreload 9236 140438883253120 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:37:39,990 autoreload 9260 140372683115392 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 20:37:59,961 log 9260 140372550350400 Internal Server Error: /customers/c7d83c33-a064-4016-a16d-4004b9120002/customer_notes_list/86
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/customers/views.py", line 151, in customer_view_notes
    reverse("customers-urls.customer_notes_list", args=[customer_token]))
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/urls/base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/urls/resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'customers-urls.customer_notes_list' not found. 'customers-urls.customer_notes_list' is not a valid view function or pattern name.
django.utils.autoreload INFO 2024-05-29 20:38:53,576 autoreload 9260 140372683115392 /workspaces/django-renterprise/customers/urls.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:38:55,865 autoreload 9376 140133145176960 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:38:59,162 autoreload 9376 140133145176960 /workspaces/django-renterprise/customers/urls.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:39:01,666 autoreload 9400 140498035035008 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:41:39,715 autoreload 9400 140498035035008 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:41:41,876 autoreload 9449 139738384415616 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:41:54,162 autoreload 9449 139738384415616 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:41:56,481 autoreload 9502 140132519099264 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:54:25,468 autoreload 9502 140132519099264 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:54:27,596 autoreload 9555 139635612904320 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:54:41,865 autoreload 9555 139635612904320 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:54:43,800 autoreload 9600 139977543961472 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:54:48,065 autoreload 9600 139977543961472 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:54:50,479 autoreload 9635 140210263804800 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:55:00,995 autoreload 9635 140210263804800 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:55:02,766 autoreload 9676 140209699355520 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:57:04,873 autoreload 9676 140209699355520 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:57:06,961 autoreload 9721 139770832259968 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:57:15,798 autoreload 9721 139770832259968 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:57:17,784 autoreload 9763 139882124041088 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:57:25,254 autoreload 9763 139882124041088 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:57:27,454 autoreload 9808 140447889935232 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:57:32,902 autoreload 9808 140447889935232 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:57:34,873 autoreload 9852 140061759679360 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:57:38,098 autoreload 9852 140061759679360 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:57:40,063 autoreload 9874 140215664970624 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:57:50,865 autoreload 9874 140215664970624 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:57:52,873 autoreload 9904 140699738762112 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:59:05,091 autoreload 9904 140699738762112 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:59:07,189 autoreload 9946 140003637029760 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:59:09,401 autoreload 9946 140003637029760 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:59:11,756 autoreload 9970 140086793137024 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 20:59:47,395 autoreload 9970 140086793137024 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 20:59:49,293 autoreload 10027 139851889286016 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:00:09,396 autoreload 10027 139851889286016 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:00:11,402 autoreload 10072 140699610483584 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:00:14,899 autoreload 10072 140699610483584 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:00:16,853 autoreload 10096 140434276076416 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:00:20,245 autoreload 10096 140434276076416 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:00:22,193 autoreload 10125 139956777839488 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:00:28,079 autoreload 10125 139956777839488 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:00:29,890 autoreload 10163 139694229056384 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:00:32,368 autoreload 10163 139694229056384 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:00:34,808 autoreload 10184 139995270683520 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:00:37,194 autoreload 10184 139995270683520 /workspaces/django-renterprise/customers/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:00:39,351 autoreload 10208 139975275096960 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:05:40,453 autoreload 10599 140073504828288 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 21:06:28,623 log 10599 140073377986112 Internal Server Error: /items/create/item_type/
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/items/views.py", line 80, in item_type_create
    item_type.name = request.name
AttributeError: 'WSGIRequest' object has no attribute 'name'
django.request WARNING 2024-05-29 21:06:29,164 log 10599 140073377986112 Not Found: /favicon.ico
django.utils.autoreload INFO 2024-05-29 21:08:45,981 autoreload 10599 140073504828288 /workspaces/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:08:48,147 autoreload 10787 140103594847104 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:08:58,011 autoreload 10787 140103594847104 /workspaces/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:09:00,245 autoreload 10830 140464873675648 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:09:04,891 autoreload 10830 140464873675648 /workspaces/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:09:06,941 autoreload 10875 139850094795648 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:09:10,268 autoreload 10875 139850094795648 /workspaces/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:09:12,085 autoreload 10899 139722615733120 Watching for file changes with StatReloader
django.request ERROR 2024-05-29 21:09:45,589 log 10899 139722556110400 Internal Server Error: /items/create/
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/items/views.py", line 52, in item_create
    item.delivery_date = null
NameError: name 'null' is not defined
django.utils.autoreload INFO 2024-05-29 21:10:13,678 autoreload 10899 139722615733120 /workspaces/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:10:15,946 autoreload 11054 139854728125312 Watching for file changes with StatReloader
django.request WARNING 2024-05-29 21:14:31,864 log 11054 139854601254464 Not Found: /items/56/styles.css.map
django.utils.autoreload INFO 2024-05-29 21:16:34,101 autoreload 11054 139854728125312 /workspaces/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:16:36,366 autoreload 11422 140232205446016 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:16:43,171 autoreload 11422 140232205446016 /workspaces/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:16:45,055 autoreload 11467 140145863699328 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:19:11,598 autoreload 11467 140145863699328 /workspaces/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:19:13,656 autoreload 11520 139896185654144 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:20:29,498 autoreload 11520 139896185654144 /workspaces/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:20:31,789 autoreload 11565 139875614747520 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:20:35,161 autoreload 11565 139875614747520 /workspaces/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:20:37,372 autoreload 11589 140247091534720 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:22:58,965 autoreload 11589 140247091534720 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:23:01,046 autoreload 11638 140355570899840 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:23:03,292 autoreload 11638 140355570899840 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:23:05,175 autoreload 11662 139959378340736 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:41:46,598 autoreload 11662 139959378340736 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:41:48,588 autoreload 11707 139861214915456 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:42:04,187 autoreload 11707 139861214915456 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:42:06,787 autoreload 11749 140615390354304 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:42:11,080 autoreload 11749 140615390354304 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:42:12,888 autoreload 11791 139702380440448 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:42:51,285 autoreload 11791 139702380440448 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:42:53,371 autoreload 11836 140288124210048 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:42:58,994 autoreload 11836 140288124210048 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:43:00,978 autoreload 11878 139955041205120 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:43:19,587 autoreload 11878 139955041205120 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:43:21,648 autoreload 11923 139812893272960 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:43:36,122 autoreload 11923 139812893272960 /workspaces/django-renterprise/env.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:55:03,989 autoreload 12172 139860240092032 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 21:55:35,559 autoreload 12172 139860240092032 /workspaces/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 21:55:37,665 autoreload 12226 140567886519168 Watching for file changes with StatReloader
django.request WARNING 2024-05-29 21:55:47,787 log 12226 140567799883328 Not Found: /favicon.ico
django.request WARNING 2024-05-29 21:55:48,128 log 12226 140567816668736 Not Found: /admin/styles.css.map
django.request WARNING 2024-05-29 21:55:49,490 log 12226 140567799883328 Not Found: /admin/styles.css.map
django.request WARNING 2024-05-29 21:56:52,444 log 12226 140567799883328 Not Found: /styles.css.map
django.request WARNING 2024-05-29 21:56:56,056 log 12226 140567799883328 Not Found: /styles.css.map
django.request WARNING 2024-05-29 21:56:56,458 log 12226 140567799883328 Not Found: /items/styles.css.map
django.request WARNING 2024-05-29 21:56:58,867 log 12226 140567799883328 Not Found: /items/styles.css.map
django.request WARNING 2024-05-29 21:56:59,248 log 12226 140567799883328 Not Found: /items/create/item_type/styles.css.map
django.request WARNING 2024-05-29 21:57:25,970 log 12226 140567799883328 Not Found: /items/create/item_type/styles.css.map
django.request WARNING 2024-05-29 21:58:14,297 log 12226 140567799883328 Not Found: /admin/styles.css.map
django.utils.autoreload INFO 2024-05-29 22:02:00,074 autoreload 12226 140567886519168 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:02:02,282 autoreload 12784 139957246495616 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:02:35,475 autoreload 12784 139957246495616 /workspaces/django-renterprise/renterprise/urls.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:02:37,485 autoreload 12831 139997885647744 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:02:40,611 autoreload 12831 139997885647744 /workspaces/django-renterprise/renterprise/urls.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:02:42,585 autoreload 12855 140699659402112 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:02:46,809 autoreload 12855 140699659402112 /workspaces/django-renterprise/renterprise/urls.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:02:48,942 autoreload 12890 140209384844160 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:02:51,083 autoreload 12890 140209384844160 /workspaces/django-renterprise/renterprise/urls.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:02:53,047 autoreload 12902 139780369574784 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:02:55,878 autoreload 12902 139780369574784 /workspaces/django-renterprise/renterprise/urls.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:02:58,293 autoreload 12926 139688189782912 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:05:11,778 autoreload 12926 139688189782912 /workspaces/django-renterprise/renterprise/urls.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:05:13,692 autoreload 12984 140227882183552 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:05:15,363 autoreload 12984 140227882183552 /workspaces/django-renterprise/renterprise/urls.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:05:17,282 autoreload 13008 139924871117696 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:05:21,381 autoreload 13008 139924871117696 /workspaces/django-renterprise/renterprise/urls.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:05:23,266 autoreload 13032 139789768780672 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:05:29,608 autoreload 13032 139789768780672 /workspaces/django-renterprise/renterprise/urls.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:05:31,856 autoreload 13074 139634358193024 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:05:38,099 autoreload 13074 139634358193024 /workspaces/django-renterprise/renterprise/urls.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:05:40,100 autoreload 13112 140421067033472 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:05:43,061 autoreload 13112 140421067033472 /workspaces/django-renterprise/renterprise/urls.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:05:45,059 autoreload 13135 140234176949120 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:08:53,405 autoreload 13135 140234176949120 /workspaces/django-renterprise/renterprise/urls.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:08:55,649 autoreload 13165 139947068615552 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:09:02,463 autoreload 13165 139947068615552 /workspaces/django-renterprise/renterprise/urls.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:09:04,601 autoreload 13210 140630261324672 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:09:06,490 autoreload 13210 140630261324672 /workspaces/django-renterprise/renterprise/urls.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:09:08,743 autoreload 13240 140059182627712 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:09:15,400 autoreload 13240 140059182627712 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:13:04,659 autoreload 13383 140021713308544 Watching for file changes with StatReloader
django.request WARNING 2024-05-29 22:13:17,568 log 13383 140021570991680 Not Found: /styles.css.map
django.request WARNING 2024-05-29 22:13:18,254 log 13383 140021562598976 Not Found: /styles.css.map
django.request WARNING 2024-05-29 22:13:21,662 log 13383 140021562598976 Not Found: /styles.css.map
django.request WARNING 2024-05-29 22:13:21,876 log 13383 140021562598976 Not Found: /items/styles.css.map
django.request WARNING 2024-05-29 22:13:24,166 log 13383 140021562598976 Not Found: /items/styles.css.map
django.request WARNING 2024-05-29 22:13:24,467 log 13383 140021562598976 Not Found: /items/create/item_type/styles.css.map
django.request WARNING 2024-05-29 22:14:01,348 log 13383 140021562598976 Not Found: /items/create/item_type/styles.css.map
django.request WARNING 2024-05-29 22:14:11,946 log 13383 140021562598976 Not Found: /items/create/styles.css.map
django.request WARNING 2024-05-29 22:14:14,955 log 13383 140021562598976 Not Found: /items/create/item_type/styles.css.map
django.utils.autoreload INFO 2024-05-29 22:17:29,550 autoreload 13383 140021713308544 /workspaces/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:17:31,997 autoreload 13654 140109641616256 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:19:28,850 autoreload 13654 140109641616256 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:19:31,401 autoreload 13703 140543593728896 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:19:47,971 autoreload 13703 140543593728896 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:19:49,961 autoreload 13748 140491467688832 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:20:35,108 autoreload 13748 140491467688832 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:20:37,589 autoreload 13790 140149558619008 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:20:44,340 autoreload 13790 140149558619008 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:20:46,880 autoreload 13835 139887153453952 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:20:51,268 autoreload 13835 139887153453952 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:20:53,665 autoreload 13870 139849754844032 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:21:09,509 autoreload 13870 139849754844032 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:21:12,145 autoreload 13922 140532408363904 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:21:16,453 autoreload 13922 140532408363904 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:21:18,569 autoreload 13966 140235997793152 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-05-29 22:23:46,892 connectionpool 13966 140235936290368 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-05-29 22:23:47,349 connectionpool 13966 140235936290368 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 404 0
django.request WARNING 2024-05-29 22:23:49,559 log 13966 140235936290368 Not Found: /items/create/item_type/styles.css.map
django.request WARNING 2024-05-29 22:24:05,159 log 13966 140235936290368 Not Found: /favicon.ico
django.request WARNING 2024-05-29 22:24:06,254 log 13966 140235936290368 Not Found: /admin/styles.css.map
django.request WARNING 2024-05-29 22:24:29,763 log 13966 140235926586944 Not Found: /items/itemtype/
django.request WARNING 2024-05-29 22:24:30,251 log 13966 140235926586944 Not Found: /items/itemtype/styles.css.map
django.request WARNING 2024-05-29 22:24:44,967 log 13966 140235936290368 Not Found: /items/item_type_create/
django.request WARNING 2024-05-29 22:24:45,365 log 13966 140235936290368 Not Found: /items/item_type_create/styles.css.map
django.request WARNING 2024-05-29 22:24:50,746 log 13966 140235936290368 Not Found: /styles.css.map
django.request WARNING 2024-05-29 22:24:54,084 log 13966 140235936290368 Not Found: /items/styles.css.map
django.request WARNING 2024-05-29 22:24:56,654 log 13966 140235936290368 Not Found: /items/create/item_type/styles.css.map
django.request ERROR 2024-05-29 22:25:09,496 log 13966 140235936290368 Internal Server Error: /items/create/item_type/
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/items/views.py", line 87, in item_type_create
    item_type.save()
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1821, in execute_sql
    for sql, params in self.as_sql():
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1745, in as_sql
    value_rows = [
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1746, in <listcomp>
    [
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1747, in <listcomp>
    self.prepare_value(field, self.pre_save_val(field, obj))
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1695, in pre_save_val
    return field.pre_save(obj, add=True)
  File "/home/cistudent/.local/lib/python3.9/site-packages/cloudinary/models.py", line 112, in pre_save
    instance_value = uploader.upload_resource(value, **options)
  File "/home/cistudent/.local/lib/python3.9/site-packages/cloudinary/uploader.py", line 64, in upload_resource
    result = upload_func(file, **options)
  File "/home/cistudent/.local/lib/python3.9/site-packages/cloudinary/uploader.py", line 45, in upload
    return call_cacheable_api("upload", params, file=file, **options)
  File "/home/cistudent/.local/lib/python3.9/site-packages/cloudinary/uploader.py", line 462, in call_cacheable_api
    result = call_api(action, params, http_headers, return_error, unsigned, file, timeout, **options)
  File "/home/cistudent/.local/lib/python3.9/site-packages/cloudinary/uploader.py", line 487, in call_api
    params = utils.sign_request(params, options)
  File "/home/cistudent/.local/lib/python3.9/site-packages/cloudinary/utils.py", line 620, in sign_request
    raise ValueError("Must supply api_secret")
ValueError: Must supply api_secret
django.request WARNING 2024-05-29 22:25:10,379 log 13966 140235936290368 Not Found: /items/create/item_type/styles.css.map
django.utils.autoreload INFO 2024-05-29 22:29:21,270 autoreload 13966 140235997793152 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:29:23,843 autoreload 14323 140621489351552 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:31:52,759 autoreload 14323 140621489351552 /workspaces/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:31:55,183 autoreload 14376 140178341149568 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:32:38,732 autoreload 14376 140178341149568 /workspaces/django-renterprise/items/views.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:32:40,961 autoreload 14453 140237340617600 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-05-29 22:32:56,067 connectionpool 14453 140237279114816 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-05-29 22:32:56,349 connectionpool 14453 140237279114816 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 404 0
django.request WARNING 2024-05-29 22:32:58,581 log 14453 140237279114816 Not Found: /items/create/item_type/styles.css.map
django.request ERROR 2024-05-29 22:33:02,498 log 14453 140237279114816 Internal Server Error: /items/create/item_type/
Traceback (most recent call last):
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/workspaces/django-renterprise/items/views.py", line 80, in item_type_create
    item_type.save()
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1821, in execute_sql
    for sql, params in self.as_sql():
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1745, in as_sql
    value_rows = [
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1746, in <listcomp>
    [
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1747, in <listcomp>
    self.prepare_value(field, self.pre_save_val(field, obj))
  File "/home/cistudent/.local/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1695, in pre_save_val
    return field.pre_save(obj, add=True)
  File "/home/cistudent/.local/lib/python3.9/site-packages/cloudinary/models.py", line 112, in pre_save
    instance_value = uploader.upload_resource(value, **options)
  File "/home/cistudent/.local/lib/python3.9/site-packages/cloudinary/uploader.py", line 64, in upload_resource
    result = upload_func(file, **options)
  File "/home/cistudent/.local/lib/python3.9/site-packages/cloudinary/uploader.py", line 45, in upload
    return call_cacheable_api("upload", params, file=file, **options)
  File "/home/cistudent/.local/lib/python3.9/site-packages/cloudinary/uploader.py", line 462, in call_cacheable_api
    result = call_api(action, params, http_headers, return_error, unsigned, file, timeout, **options)
  File "/home/cistudent/.local/lib/python3.9/site-packages/cloudinary/uploader.py", line 487, in call_api
    params = utils.sign_request(params, options)
  File "/home/cistudent/.local/lib/python3.9/site-packages/cloudinary/utils.py", line 620, in sign_request
    raise ValueError("Must supply api_secret")
ValueError: Must supply api_secret
django.request WARNING 2024-05-29 22:33:03,480 log 14453 140237279114816 Not Found: /items/create/item_type/styles.css.map
django.utils.autoreload INFO 2024-05-29 22:36:42,488 autoreload 14453 140237340617600 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:36:44,845 autoreload 14585 139915735862144 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:36:53,565 autoreload 14585 139915735862144 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:36:56,244 autoreload 14630 140694597598080 Watching for file changes with StatReloader
django.utils.autoreload INFO 2024-05-29 22:36:58,481 autoreload 14630 140694597598080 /workspaces/django-renterprise/renterprise/settings.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:37:01,088 autoreload 14654 140672245001088 Watching for file changes with StatReloader
urllib3.connectionpool DEBUG 2024-05-29 22:37:53,663 connectionpool 14654 140672183498304 Starting new HTTPS connection (1): res.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-05-29 22:37:53,723 connectionpool 14654 140672183498304 https://res.cloudinary.com:443 "GET /dddubflsq/raw/upload/v1/static/staticfiles.json HTTP/1.1" 404 0
django.request WARNING 2024-05-29 22:37:55,518 log 14654 140672183498304 Not Found: /items/create/item_type/styles.css.map
urllib3.connectionpool DEBUG 2024-05-29 22:38:02,909 connectionpool 14654 140672183498304 Starting new HTTPS connection (1): api.cloudinary.com:443
urllib3.connectionpool DEBUG 2024-05-29 22:38:03,797 connectionpool 14654 140672183498304 https://api.cloudinary.com:443 "POST /v1_1/dddubflsq/image/upload HTTP/1.1" 200 None
django.request WARNING 2024-05-29 22:38:04,842 log 14654 140672183498304 Not Found: /items/create/item_type/styles.css.map
django.request WARNING 2024-05-29 22:38:13,578 log 14654 140672183498304 Not Found: /admin/styles.css.map
django.utils.autoreload INFO 2024-05-29 22:38:40,800 autoreload 14654 140672245001088 /workspaces/django-renterprise/env.py changed, reloading.
django.utils.autoreload INFO 2024-05-29 22:38:43,440 autoreload 14869 139641968941952 Watching for file changes with StatReloader
